### ISSUE TITLE
Configuration for rule "react/jsx-handler-names" is invalid

### ISSUE THREAD

**Issue by daviddelusenet** [yannickcr/eslint-plugin-react#2498]
When I add the following part to my `.eslintrc.js`:

```js
'react/jsx-handler-names': ['error', {
    eventHandlerPrefix: 'handle',
    eventHandlerPropPrefix: 'on',
    checkLocalVariables: false,
}],
```

The following error gets thrown:

```bash
Error: --config:
	Configuration for rule "react/jsx-handler-names" is invalid:
	Value {"eventHandlerPrefix":"handle","eventHandlerPropPrefix":"on","checkLocalVariables":false} should NOT have additional properties.
	Value {"eventHandlerPrefix":"handle","eventHandlerPropPrefix":"on","checkLocalVariables":false} should NOT have additional properties.
	Value {"eventHandlerPrefix":"handle","eventHandlerPropPrefix":"on","checkLocalVariables":false} should NOT have additional properties.
	Value {"eventHandlerPrefix":"handle","eventHandlerPropPrefix":"on","checkLocalVariables":false} should match exactly one schema in oneOf.
```

ESLint: `6.6.0`
ESLint plugin React: `7.16.0`

Please let me know if you need any more information.

**Comment by ljharb** [comment]
Seems like you're reading the docs from master, which is unreleased; always check the docs from the proper version tag: https://github.com/yannickcr/eslint-plugin-react/blob/v7.16.0/docs/rules/jsx-handler-names.md

**Comment by daviddelusenet** [comment]
Thanks for the reply @ljharb !
